rs/replicated_state/src/canister_state/system_state.rs

```diff
@@ -265,6 +268,206 @@ impl CanisterHistory {
     }
 }
 
+#[derive(Clone, Eq, PartialEq, Debug, Default)]
+pub struct TaskQueue {
+    /// Tasks to execute before processing input messages.
+    /// Currently the task queue is empty outside of execution rounds.
```

Comment:
You are right, this is outdated comment. That part is not correct. Even the first part of the comment is not true anymore, because we alternate between messages and tasks to be executed first on every round in the current implementation.